tests/assets/test_divisible_assets.py

```diff
@@ -0,0 +1,686 @@
+    assert len(tx_signed.fulfillments) == 1
+
+
+# CREATE divisible asset
+# Single input
+# Single owners_before
+# Single output
+# Multiple owners_after
+def test_single_in_single_own_single_out_multiple_own_create(b, user_vk):
+    from bigchaindb.models import Transaction
+    from bigchaindb.common.transaction import Asset
+
+    asset = Asset(divisible=True)
+    tx = Transaction.create([b.me], [([user_vk, user_vk], 100)], asset=asset)
+    tx_signed = tx.sign([b.me_private])
+
+    assert tx_signed.validate(b) == tx_signed
+    assert len(tx_signed.conditions) == 1
+    assert tx_signed.conditions[0].amount == 100
+
+    condition = tx_signed.conditions[0].to_dict()
+    assert 'subfulfillments' in condition['condition']['details']
+    assert len(condition['condition']['details']['subfulfillments']) == 2
+
+    assert len(tx_signed.fulfillments) == 1
+
+
+# CREATE divisible asset
+# Single input
+# Single owners_before
+# Multiple outputs
+# Mix: one output with a single owners_after, one output with multiple
+#      owners_after
+def test_single_in_single_own_multiple_out_mix_own_create(b, user_vk):
+    from bigchaindb.models import Transaction
+    from bigchaindb.common.transaction import Asset
+
+    asset = Asset(divisible=True)
+    tx = Transaction.create([b.me],
+                            [([user_vk], 50), ([user_vk, user_vk], 50)],
+                            asset=asset)
+    tx_signed = tx.sign([b.me_private])
+
+    assert tx_signed.validate(b) == tx_signed
+    assert len(tx_signed.conditions) == 2
+    assert tx_signed.conditions[0].amount == 50
+    assert tx_signed.conditions[1].amount == 50
+
+    condition_cid1 = tx_signed.conditions[1].to_dict()
+    assert 'subfulfillments' in condition_cid1['condition']['details']
+    assert len(condition_cid1['condition']['details']['subfulfillments']) == 2
+
+    assert len(tx_signed.fulfillments) == 1
+
+
+# CREATE divisible asset
+# Single input
+# Multiple owners_before
+# Ouput combinations already tested above
+def test_single_in_multiple_own_single_out_single_own_create(b, user_vk,
+                                                             user_sk):
+    from bigchaindb.models import Transaction
+    from bigchaindb.common.transaction import Asset
+
+    asset = Asset(divisible=True)
+    tx = Transaction.create([b.me, user_vk], [([user_vk], 100)], asset=asset)
+    tx_signed = tx.sign([b.me_private, user_sk])
+    assert tx_signed.validate(b) == tx_signed
+    assert len(tx_signed.conditions) == 1
+    assert tx_signed.conditions[0].amount == 100
+    assert len(tx_signed.fulfillments) == 1
+
+    ffill = tx_signed.fulfillments[0].fulfillment.to_dict()
+    assert 'subfulfillments' in ffill
+    assert len(ffill['subfulfillments']) == 2
+
+
+# TRANSFER divisible asset
+# Single input
+# Single owners_before
+# Single output
+# Single owners_after
+# TODO: I don't really need inputs. But I need the database to be setup or
```

Comment:
Create conftest setup for this?

I think if we don't do it now, we'll probably never do it.